Gemfile.lock

```diff
@@ -8,56 +8,56 @@ GIT
 
 GIT
   remote: https://github.com/guidance-guarantee-programme/princely.git
-  revision: 4ea8b6be15257b5afc3cfdb1767b8d8c75c72750
+  revision: 9b7170d0350e537da4491958f1977ee1a514d34b
   branch: remove-alias-method-chain
   specs:
-    princely (2.0.2)
+    princely (2.0.3)
```

Comment:
I did take a look into that it it looks as though the main (mbleigh) repo's been somewhat forgotten about with plenty of forks, so I didn't dig much deeper.

I'll pull it down tomorrow and get a clearer picture of the situation rather than trying to work it all out from the github network graph.